### PR TITLE
Update config doc on the grafeas note part.

### PR DESCRIPTION
**Pull request by chuangw6**
- Added a link to Grafeas doc about notes and occurrences.
- Explained a bit more on how we will use the configurable field
to create notes.

### CODE REVIEW

**Comment by chuangw6** [comment]
/assign @wlynch
PTAL. Thanks

**Review by wlynch — CHANGES_REQUESTED** [review]
Totally forgot to hit submit on this yesterday 🤦

File: docs/config.md [hunk 1]
```diff
@@ -76,18 +76,17 @@ Supported keys include:
 | `storage.gcs.bucket` | The GCS bucket for storage | | |
 | `storage.oci.repository` | The OCI repo to store OCI signatures in  | | |
 | `storage.docdb.url` | The go-cloud URI reference to a docstore collection | `firestore://projects/[PROJECT]/databases/(default)/documents/[COLLECTION]?name_field=name`| |
-|`storage.grafeas.projectid`|The project ID to store occurrences|||
-|`storage.grafeas.noteid` (optional)|The note ID to link occurrences. If noteid is not provided, a name in the format of `tekton-<NAMESPACE>` will be used.|||
+|`storage.grafeas.projectid`|The project of where grafeas server is located for storing occurrences|||
+|`storage.grafeas.noteid` (optional)|This field will be used as the prefix part of the note name that will be created. The value of this field must be a string without spaces. (See more details below.) |||
```
[the remark this answers is not in the document]
Comment (chuangw6) [reply docs/config.md]: SG! Added!

**Review by wlynch — APPROVED** [review]

**Pull request merged**